src/madhuraMJun20/assignment30Plus/CodingExam_12.java

```diff
@@ -0,0 +1,25 @@
+package madhuraMJun20.assignment30Plus;
+/*Find triplet having maximum sum.
+int[] num = {10,20,12,22,15,13,9,17,21}
+Max sum triplet is : 22,15,13 -> 50*/
+public class CodingExam_12 {
+	void sumOfTriplets(int[] num) {
+		int sum = 0;
+		int maxSum = 0;
+		String triplet = "";
+		for (int index = 0; index < num.length - 2; index++) {
+			sum = num[index] + num[index + 1] + num[index + 2];
```

Comment:
Hi @MadhuraMJun20 
Improvement Point : you can add index+=3 instead of index++ (online 16th index = index + 2) handle by index+=3
still very well work

Reply:
Ok.. I will remember that now onwards..Thank you..!